<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-47P2MTB" failures="2" tests="28" name="test.SwagLabs1LoginTest" time="185.958" errors="0" timestamp="2023-03-02T19:35:17 IST" skipped="0">
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="11.626"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="6.025"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="5.039">
    <failure message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at test.SwagLabs1LoginTest.doLogin_1ValidId2ValidPwd_OpenedHomepage(SwagLabs1LoginTest.java:72)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- doLogin_1ValidId2ValidPwd_OpenedHomepage -->
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="9.820"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="6.755">
    <failure message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at test.SwagLabs1LoginTest.doLogin_1ValidId2ValidPwd_OpenedHomepage(SwagLabs1LoginTest.java:72)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- doLogin_1ValidId2ValidPwd_OpenedHomepage -->
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="5.869"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="7.174"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="5.944"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1ValidId2InvalidPwd_FailedLoginAndVerifyErrorMSG" time="9.689"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1ValidId2InvalidPwd_FailedLoginAndVerifyErrorMSG" time="5.319"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1InvalidId2validPwd_FailedLoginAndVerifyErrorMSG" time="6.081"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1InvalidId2validPwd_FailedLoginAndVerifyErrorMSG" time="5.410"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1InvalidId2InvalidPwd_FailedLoginAndVerifyErrorMSG" time="5.659"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1InvalidId2InvalidPwd_FailedLoginAndVerifyErrorMSG" time="6.050"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1EmptyId2GivenPwd_FailedLoginAndVerifyWarningMSG" time="5.825"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1EmptyId2GivenPwd_FailedLoginAndVerifyWarningMSG" time="4.787"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1EmptyPwd2GivenId_FailedLoginAndVerifyWarningMSG" time="6.750"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1EmptyPwd2GivenId_FailedLoginAndVerifyWarningMSG" time="5.044"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1EmptyId2EmptyPwd_FailedLoginAndVerifyWarningMSG" time="6.684"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_1EmptyId2EmptyPwd_FailedLoginAndVerifyWarningMSG" time="7.610"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_Clickable_CompletedLogin" time="6.326"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_Clickable_CompletedLogin" time="11.034"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_SwagLabsLogo_DisplayedLogoAndVerifyText" time="5.351"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_SwagLabsLogo_DisplayedLogoAndVerifyText" time="6.373"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_ErrorBoxCancelButton_ClosedErrorOrWarningMSG" time="7.598"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_ErrorBoxCancelButton_ClosedErrorOrWarningMSG" time="6.188"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_PlaceHolderText_Displayed" time="5.587"/>
  <system-out/>
  <testcase classname="test.SwagLabs1LoginTest" name="doLogin_PlaceHolderText_Displayed" time="4.341"/>
  <system-out/>
</testsuite> <!-- test.SwagLabs1LoginTest -->
